Service/EndpointService.php

```diff
@@ -208,7 +208,7 @@ public function getEndpoint(): Endpoint
         $path = substr($path, 5);
         $endpoint = $this->entityManager->getRepository('App:Endpoint')->findByMethodRegex($this->request->getMethod(), $path);
 
-        if ($endpoint !== null) {
+        if ($endpoint instanceof Endpoint === true) {
```

Comment:
This feels a bit contradictory with the change in CacheService, any special reason to take the more complex route here?

Reply:
I wasn't sure if findByMethodRegex returned null or not and at some point i changed all checks to instanceof checks, only didn't change this one back :P

Reply:
Checked and it should result Endpoint or null, so will do a null check here as well